### DISCUSSION TITLE
Two Months Without Support for Blocked Accounts—Urgent Resolution Needed

### DISCUSSION THREAD

**Discussion by Erfan-mm** [community/community#143462] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

General

### Body

I’m writing on behalf of my friend, whose GitHub account was unexpectedly marked as spam and subsequently blocked. We immediately submitted a support ticket over two months ago, but to date, we have received no response or update from the GitHub support team.

The lack of communication and resolution is incredibly frustrating and concerning, particularly given how crucial GitHub profiles are for developers. Many of us rely on GitHub not only for version control and collaboration but also as a professional portfolio, linking repositories in resumes and CVs to showcase skills and projects to potential employers. When an account is blocked, this work becomes hidden, affecting not only visibility but potentially jeopardizing future career opportunities.

Reliability and support are foundational to any platform, and it's disheartening when issues like these are left unaddressed. We understand that GitHub must enforce rules to prevent misuse, but it is equally important to provide a clear path for users to resolve issues and regain access if flagged in error.

If there is any way to escalate this matter or if someone could provide guidance on steps we can take to speed up the review, it would be greatly appreciated. We hope to resolve this soon and restore access to a valuable and essential professional account.

@github-staff

account: https://github.com/erfanmomeniii
ticket: https://support.github.com/ticket/personal/0/2979214 

![Screenshot (1118)](https://github.com/user-attachments/assets/48c47cef-2960-4a21-be9f-81eaa9107af1)

**Comment by M-tech-cmd** [comment]
Follow Up on the Ticket: Make sure to follow up on your support ticket through the GitHub Support portal. Sometimes, additional follow-ups can help prioritize your request.

Contact GitHub Support via Social Media: Reach out to GitHub Support on Twitter or other social media platforms. Publicly mentioning your issue can sometimes expedite the process.

Community Forums: Engage with the GitHub Community forums where you posted your issue. Other users or GitHub staff might provide additional insights or assistance.

Email Support: If you haven’t already, try emailing GitHub Support directly with all relevant details and your ticket number.

**Reply by Erfan-mm** [reply]
Thank you for the suggestions.

Regarding the steps you mentioned:

1. Ticket Follow-Up: We have added additional comments to the existing support ticket, and we even created a new ticket over two weeks ago. However, we have yet to receive any responses from GitHub Support.

2. Social Media Outreach: We reached out to multiple GitHub Support engineers on LinkedIn, but unfortunately, while the messages were seen, we did not receive any replies.

3. Community Forums: Currently, I’m the only one able to create discussions in the GitHub Community forums, so this route is not accessible for the friend who needs assistance and blocked his account.

4. Email Contact: It appears that GitHub Support does not provide a direct email for inquiries, which limits our ability to reach them outside the support portal.

Thank you again for your advice—please let me know if there are any additional approaches you recommend.

**Comment by radanpro** [comment]
Check these:

Enable Two-Factor Authentication (2FA):
   - Ensure that the account has two-factor authentication enabled. This can help enhance the security of the account and might expedite the review process.

Submit a Reinstatement Request:
   - You can submit a request to reinstate the account via this link: https://support.github.com/contact/reinstatement. Make sure to explain the issue clearly and mention all the steps you have taken so far.

 Contact GitHub Support:

**Comment by devaapriana** [comment]
Looks like your account has been restored, what date did they respond to your ticket?
